jellyfin.tf

```diff
@@ -312,20 +291,41 @@ EOF
  *   if you've provided a domain.
  */
 
-resource "aws_default_vpc" "default" { }
+resource "aws_vpc" "jellyfin_vpc" { 
```

Comment:
Nice.

Reply:
Similarly, thought, I would use a `Name` tag so it's easily identifiable on the AWS dashboard.

Reply:
Added the tags